docs/model_endpoint_management.ipynb

```diff
@@ -0,0 +1,572 @@
+    "* model_auth_id (Optional[str]) : The secret ID that you set and is subsequently used when registering a model endpoint.\n",
+    "* input_transform_fn (Optional[str]) : The SQL function name to transform input of the corresponding prediction function to the model-specific input.\n",
+    "* output_transform_fn (Optional[str]) : The SQL function name to transform model specific output to the prediction function output."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Generate Vector Embeddings with `AlloyDBEmbeddings`\n",
+    "The `AlloyDBEmbeddings` class allows users to utilize the AlloyDB Embeddings available via Model Endpoint Management.\n",
+    "\n",
+    "In the below example, we are using the `textembedding-gecko@003` model that we have created using the Model Manager."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note**: This tutorial demonstrates the async interface. All async methods have corresponding sync methods."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBEmbeddings\n",
+    "\n",
+    "model_id = \"textembedding-gecko@003\"\n",
+    "embedding_service = await AlloyDBEmbeddings.create(engine=engine, model_id=model_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "On creating an instance of the `AlloyDBEmbeddings` class, it checks if the model exists.\n",
+    "If the model does not exist with that model_id, the class throws a `ValueError`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Using AlloyDBEmbeddings as an embedding service"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `AlloyDBEmbeddings` class can be used as the embedding service with an `AlloyDBVectorStore`.\n",
+    "\n",
+    "Learn more about getting started with the [`AlloyDBVectorStore`](https://github.com/googleapis/langchain-google-alloydb-pg-python/blob/main/docs/vector_store.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the below code snippet, we create a new vector store table in the DB. The `AlloyDBVectorStore` instance uses this table.\n",
+    "\n",
+    "The `aadd_documents()` function takes in prepared texts and uuids as their primary key, creates an embedding for each text using the `AlloyDBEmbeddings` instance and adds it into the vector store table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBVectorStore\n",
```

Comment:
I think we can simplify without the additional text. Here I better group the code and dependencies to make this example more clear. 

```
import uuid
from langchain_core.documents import Document
from langchain_google_alloydb_pg import AlloyDBVectorStore


# (Optional) Create a new vector store table
VECTOR_SIZE = 768  # For textembeddding-gecko@003 model
await engine.ainit_vectorstore_table(
    table_name="vector_store_table",
    vector_size=VECTOR_SIZE,
    overwrite_existing=True,
)

# Initialize the vector store instance with AlloyDBEmbeddings
vs = await AlloyDBVectorStore.create(
    engine,
    embedding_service=embedding_service,
    table_name="vector_store_table",
)

# Add documents 
texts = ["foo", "bar", "baz", "boo"]
ids = [str(uuid.uuid4()) for i in range(len(texts))]
docs = [
    Document(page_content=texts[i]) for i in range(len(texts))
]
await vs.aadd_documents(docs, ids=ids)

# Search documents 
results = await vs.asimilarity_search("foo")
```